packages/core/components/RichTextEditor/editor-styles.css

```diff
@@ -0,0 +1,54 @@
+.ProseMirror {
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  cursor: text;
+}
+
+.rich-text-editor-inline .ProseMirror-focused {
+  margin: 8px;
+}
+
+.rich-text-editor .ProseMirror {
+  background: var(--puck-color-white);
+  border-width: 1px;
+  border-style: solid;
+  border-color: var(--puck-color-grey-09);
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-family: inherit;
+  font-size: 16px;
+  padding: 12px 15px;
+  transition: border-color 50ms ease-in;
+  width: 100%;
+  max-width: 100%;
+}
+
+.rich-text-editor .ProseMirror-focused {
+  border-color: var(--puck-color-grey-05);
+  outline: 2px solid var(--puck-color-azure-05);
+  transition: none;
+}
+
+.ProseMirror-focused {
+  outline: none;
+}
+
+blockquote {
+  margin: 1em 0;
+  padding: 0 1em;
+  border-left: 4px solid var(--puck-color-grey-09);
+}
+
+code {
+  background-color: var(--puck-color-grey-11);
+  padding: 0.2em 0.4em;
+  border-radius: 0.2em;
+}
+
+p:empty::before {
+  content: "\00a0"; /* non-breaking space */
+}
+
+* {
+  white-space: pre-wrap;
+}
```

Comment:
These are global styles and will affect everything in the page. At the very least, we need to scope these underneath `.rich-text-editor`. We should also probably drop these from rendering within `inline` entirely - the user shouldn't need to depend on Puck styles when rendering the final page.